<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerInfoObject</class>
 <widget class="QWidget" name="PlayerInfoObject">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>260</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>260</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>260</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QComboBox" name="cat1Box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cat2Box">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="cat3Box">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>180</y>
     <width>101</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="cat1Button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>210</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cat2Button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>210</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Card</string>
   </property>
  </widget>
  <widget class="QPushButton" name="cat3Button">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>210</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Card</string>
   </property>
  </widget>
  <widget class="QListWidget" name="cardsList">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>20</y>
     <width>151</width>
     <height>141</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="resizeMode">
    <enum>QListView::Adjust</enum>
   </property>
  </widget>
  <widget class="QTextEdit" name="renameText">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>104</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="renameButton">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Rename</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="numCardsBox">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>120</y>
     <width>42</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="numCardsLabel">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>120</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Number of Cards:</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
